recommender/docs/index.rst

```diff
@@ -0,0 +1,83 @@
+Python Client for Recommender API (`Alpha`_)
+============================================
+
+`Recommender API`_:
+
+- `Client Library Documentation`_
+- `Product Documentation`_
+
+.. _Alpha: https://github.com/googleapis/google-cloud-python/blob/master/README.rst
+.. _Recommender API: https://cloud.google.com/recommender
+.. _Client Library Documentation: https://googleapis.github.io/google-cloud-python/latest/recommender/usage.html
+.. _Product Documentation:  https://cloud.google.com/recommender
+
+Quick Start
+-----------
+
+In order to use this library, you first need to go through the following steps:
+
+1. `Select or create a Cloud Platform project.`_
+2. `Enable billing for your project.`_
+3. `Enable the Recommender API.`_
+4. `Setup Authentication.`_
+
+.. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
+.. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
+.. _Enable the Recommender API.:  https://cloud.google.com/recommender
+.. _Setup Authentication.: https://googleapis.github.io/google-cloud-python/latest/core/auth.html
+
+Installation
+~~~~~~~~~~~~
+
+Install this library in a `virtualenv`_ using pip. `virtualenv`_ is a tool to
+create isolated Python environments. The basic problem it addresses is one of
+dependencies and versions, and indirectly permissions.
+
+With `virtualenv`_, it's possible to install this library without needing system
+install permissions, and without clashing with the installed system
+dependencies.
+
+.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/
+
+
+Mac/Linux
+^^^^^^^^^
+
+.. code-block:: console
+
+    pip install virtualenv
+    virtualenv <your-env>
+    source <your-env>/bin/activate
+    <your-env>/bin/pip install google-cloud-recommender
+
+
+Windows
+^^^^^^^
+
+.. code-block:: console
+
+    pip install virtualenv
+    virtualenv <your-env>
+    <your-env>\Scripts\activate
+    <your-env>\Scripts\pip.exe install google-cloud-recommender
+
+Next Steps
+~~~~~~~~~~
+
+-  Read the `Client Library Documentation`_ for Recommender API
+   API to see other available methods on the client.
+-  Read the `Recommender API Product documentation`_ to learn
+   more about the product and see How-to Guides.
+-  View this `repository’s main README`_ to see the full list of Cloud
+   APIs that we cover.
+
+.. _Recommender API Product documentation:  https://cloud.google.com/recommender
+.. _repository’s main README: https://github.com/googleapis/google-cloud-python/blob/master/README.rst
+
+Api Reference
+-------------
+.. toctree::
+    :maxdepth: 2
+
+    gapic/v1beta1/api
+    gapic/v1beta1/types
```

Comment:
This file should be DRY'ed out (using `.. include:: README.rst`) rather than repeating the contents of the top-level `README.rst`.